frontend/src/components/ListingSection/listing-section.scss

```diff
@@ -0,0 +1,17 @@
+.listing-section-container .container {
+  display: flex;
+  padding-top: 1rem;
+  padding-bottom: 1rem;
+  // color: var(--we-grey-DETROIT);
+  max-height: 1000px;
+  font-size: 13px;
+}
+
+.flexbox-container {
+  display: -ms-flex;
+  display: -webkit-flex;
+  display: flex;
```

Comment:
The Flexbox layout model is pretty much everywhere right now: https://www.caniuse.com/#feat=flexbox
Probably no need to support really old browsers.